static/js/settings_user_groups.js

```diff
@@ -255,6 +208,60 @@ exports.populate_user_groups = function () {
     });
 };
 
+exports.set_raw_description = function (target, destination) {
```

Comment:
Renaming here as well `destination`, i.e. `group_description_element` and `target` to `target_user_group`

Reply:
Added it :+1: